Cubical/Data/Group/Base.agda

```diff
@@ -0,0 +1,22 @@
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Data.Group.Base where
+
+open import Cubical.Core.Prelude
+
+record isGroup {ℓ} (A : Set ℓ) : Set ℓ where
+  constructor group-struct
+  field
+    e  : A
+    _⁻¹  : A → A
+    _⊙_   : A → A → A
+    lUnit : ∀ a → e ⊙ a ≡ a
```

Comment:
Shouldn't there be an `rUnit` as well?

Reply:
the right ones can be derived from the left ones:
https://github.com/HoTT/HoTT-Agda/blob/1037d82edcf29b620677a311dcfd4fc2ade2faa6/core/lib/types/Group.agda#L33

But I agree this might not worth the unsymmetry.

Reply:
Oh, I see. I guess I'm more used to monoids. If it's derivable I'd skip it, I guess. If Agda had a facility for letting someone pick either one, that might be nice, but it doesn't really.

Reply:
I feel something like that can be done via type classes, but I always think that's often abused in projects to the point of ambiguity and should be used with caution.

Reply:
Its similar to how you can always adjointify equivalences but sometimes you want to give it explicitly. I think we should have the group with more axioms but add helper build functions.

Reply:
I agree with @Alizter